packages/seacas/libraries/ioss/src/Ioss_EntityBlock.C

```diff
@@ -79,3 +79,44 @@ Ioss::Property Ioss::EntityBlock::get_implicit_property(const std::string &my_na
 
   return Ioss::GroupingEntity::get_implicit_property(my_name);
 }
+
+bool Ioss::EntityBlock::equal(const Ioss::EntityBlock &rhs)
```

Comment:
Why is this needed?

Reply:
Addressed in branch `tkordenbrock-seacas/add-ioshell-compare`.